src/assets/styles/_align.scss

```diff
@@ -0,0 +1,20 @@
+body, html {
+  position: relative;
+  overflow: hidden;
+}
+
+  .v-mid,.v-m {
+    vertical-align: middle !important
```

Comment:
Ensure there is a semi-colon (;) after every property in the class. Make the required changes in all files necessary.